src/components/FlexibleForm/Control.js

```diff
@@ -357,7 +356,10 @@ export const Control = memo(props => {
     }
 
     return (
-      <Cmp {...attribs} />
+      <Cmp
+        className={classes}
```

Comment:
This line brokes styles for the <TextArea> and <Select> components. I've already fixed it before.